doc/conf.py

```diff
@@ -0,0 +1,119 @@
+# -*- coding: utf-8 -*-
+
+from os import environ
+from pathlib import Path
+from json import loads as json_loads
+from yaml import load as yaml_load, Loader as yaml_Loader
+from tabulate import tabulate
+
+ROOT = Path(__file__).resolve().parent
+
+
+# -- Generate 2019.inc and 2019/*.rst ----------------------------------------------------------------------------------
+
+with open(ROOT.parent / "issues/LCS2019.yml", "r", encoding="utf-8") as stream:
+    LCS2019 = yaml_load(stream, Loader=yaml_Loader)
+
+(ROOT / '2019').mkdir(exist_ok=True)
+
+with open(ROOT / "2019.inc", "w", encoding="utf-8") as index_fptr:
+    index_fptr.write('''\
+.. toctree::
+  :caption: 2019
+  :hidden:
+
+''')
+
+    for key, val in LCS2019.items():
+        index_fptr.write(f'  2019/{key}\n')
+        with open(ROOT / f"2019/{key}.rst", "w", encoding="utf-8") as item_fptr:
+            item_fptr.write(f'''\
+{key}
```

Comment:
@umarcor this solution is ok, but not nice :).

I believe, I have now the knowledge to create Sphinx extensions handling such cases. OTOH, it would be a dedicated extension just for that special case.

See: https://github.com/VHDL/VHDLDomain/blob/dev/VHDLDomain/Directive.py?ts=2#L457-L514

Reply:
Sure! This is just a quick and dirty solution to showcase how to use `issues/LCS2019.yml` in the doc. We are not even using all the fields yet, just the title and the body. We should improve it and use resources from pyGHDL-pyVHDLModel-VHDLModel to include VHDL content in the doc. Moreover, we might show results from CI, so that users can know which features are supported by open source simulators. We now have two open source tools than can deal with an acceptable portion of VHDL 2008 and already looking towards 2019: GHDL and NVC.
Nevertheless, we can do those improvements in upcoming PRs, so I'm merging this now.